src/client/datascience/kernel-launcher/remoteKernelFinder.ts

```diff
@@ -55,22 +55,41 @@ export class RemoteKernelFinder implements IRemoteKernelFinder {
     public async findKernel(
         resource: Resource,
         connInfo: INotebookProviderConnection | undefined,
-        option?: nbformat.INotebookMetadata | PythonEnvironment,
+        notebookMetadata?: nbformat.INotebookMetadata,
```

Comment:
Just as in `localKernelFinder`, the PytohnEnvironment wasn't being passed, I think we just missed this. if not the ts compiler will bomb out.